### ISSUE TITLE
Loss implementation different in code & paper

### ISSUE THREAD

**Issue by symphonylyh** [aim-uofa/DyCo3D#10]
Hi, similar to #9 , I noticed the types of loss mentioned in the paper (L_ctr, L_mask, L_dice) are not implemented/used in the released code. Instead, the code is using exactly the same loss as PointGroup (semantic loss, offset loss, score loss).

When I checked the value in mask_logits, I found it's almost random values that are much smaller than the threshold=0.5. Therefore, I think the current code is NOT the right version that we can see the effect of the mask head design.

Could you please take a look and update the correct version? Thanks

**Comment by tonghe90** [comment]
Hi, I have updated the code and add dice loss.

**Comment by symphonylyh** [comment]
I guess we should do loss += dice_loss in the line after? [pointgroup.py:845](https://github.com/aim-uofa/DyCo3D/blob/03b027a2924f6a0fdc717a80220c56d5ab26bccb/model/pointgroup/pointgroup.py#L845)

**Comment by tonghe90** [comment]
@symphonylyh Thanks for pointing it out. Already fixed. But As in #9, I suggest you fine-tuned the model with a ckpt that has been trained with semantic segmentation.

**Comment by ngoductuanlhp** [comment]
When adding the dice loss, I can reproduce the same results reported in the paper. Thank you @tonghe90

**Comment by tonghe90** [comment]
@ngoductuanlhp good to hear. I will close the issue for now.
